_gsocproposals/2023/proposal_GNN_tracking_object_condensation.md

```diff
@@ -94,6 +94,26 @@ Example outcomes for area 2:
 * Kilian Lieret [kl5675@princeton.edu](mailto:kl5675@princeton.edu) Princeton University
 * Gage deZoort [jdezoort@princeton.edu](mailto:jdezoort@princeton.edu) Princeton University
 
+Please see the instructions below for the phase 1 application.
+
+## Phase 1 Application
+
+Please send us your CV and motivation for this project.
+
+We also have a qualification task to make a fair selection of candidates. It should take you no more than 1-2 days to complete (depending on prior knowledge). 
+
+**For fairness, you will have 48h to send us your solutions after you receive the instructions.** 
+
+We understand that your schedule might be busy, so we proceed as follows:
+
+1. Send us a start date and time **in Eastern Time zone (EST)**. Please let us know **1-2 working days in advance** to ensure we can schedule the email in time. Your submission should be completed by Mar 12, so the **latest start date is Mar 10**.
+2. We will send a quick confirmation email within 1-2 working days.
+3. The email with the qualification task instructions will be sent at the time specified in step 1.
+4. You work for 1-2 days and submit your results (we will take into account if you submit after 1 or 2 days)
```

Comment:
Do you mean to imply that those who submit after 2 days will be 'graded' lower than those who submit after 1 day? Or do you mean that a late submission, after 2 days, will be negatively graded, but 1 day or 2 days is the same?

Version 1
```suggestion
4. You work for 1-2 days and submit your results (we may take into account the speed with which you complete the task)
```

Version 2
```suggestion
4. You work for 1-2 days and submit your results (we will penalize submissions received after 2 days)
```

Reply:
Ah, I mainly meant that 1 day or 2 day submissions are evaluated differently